cg/src/utils/blockchain.js

```diff
@@ -188,12 +199,10 @@ async function listenerForConsentEvent(callback) {
       winston.error(`Error handling consent given to ${error.toString()}`);
       return;
     }
-    if (
+    if (data.returnValues.processorsConsented) {
```

Comment:
.map was being invoked in a situation where the processorsConsented could be null, this is a simple fix for that